### ISSUE TITLE
[REPEATER]Not able to re-sort fields

### ISSUE THREAD

**Issue by plexmaniax** [dachcom-digital/pimcore-formbuilder#231]
It`s not possible to re-sort the fields in a repeater by drag n drop.

![www_voigt_ch____Pimcore](https://user-images.githubusercontent.com/18365319/88369712-e3429e00-cd90-11ea-970d-f6f52de1521a.png)

**Comment by justbane** [comment]
Yes - we are experiencing the same issue within Fieldsets... as well, we are not able to move fields into a container if they are outside of it, to begin with.

**Comment by solverat** [comment]
- Allow D&D into container
- Allow D&D out of container
- Allow Field Sorting in Container

**Comment by samowitsch** [comment]
Hi. I tried formbuilder for the first time in pimcore and had the same problem like mentioned here.
It is not possible to reorder elements inside fieldset containers. Is there any progress on this?
